nxdrive/client/uploader/direct_transfer.py

```diff
@@ -69,6 +71,11 @@ def upload(
             item = self.remote.upload_folder(
                 doc_pair.remote_parent_path,
                 {"title": doc_pair.local_name},
+                headers={
+                    REQUEST_METRICS: json.dumps(
+                        {"directTransfer.session.number": str(doc_pair.session)}
```

Comment:
To ease maintenance, we should move `directTransfer.session.number` to a contant `_SESSION_NUMBER` at the top level of that file. Do the same for all metrics.